app/controllers/cepa_health/health_controller.rb

```diff
@@ -0,0 +1,27 @@
+module CepaHealth
+  class HealthController < ApplicationController
+    before_action :check_key
+
+    def check
+      @result = CepaHealth.execute(only: params[:only], except: params[:except])
```

Comment:
Would it be paranoid to use strong_parameters here to ensure `params[:only]` etc are strings, not nested structures? i.e. `params.permit(:key, :only, :except)`